internal/pkg/test/privilege_linux.go

```diff
@@ -24,7 +24,7 @@ var origHome, unprivHome string
 func EnsurePrivilege(t *testing.T) {
 	uid := os.Getuid()
 	if uid != 0 {
-		t.Fatal("test must be run with privilege")
+		t.Skip("test must be run with privilege")
```

Comment:
this one should not affect CI as we are already ensuring that things run in the expected way, but I find it annoying that in a directory with multiple tests, they fail immediately because of lack of privilege. I rather have the test that can run do so, and the ones that require privilege to report that fact.

At any rate, undoing this is easy if you feel strongly about it:

```suggestion
		t.Fatal("test must be run with privilege")
```

Reply:
I'm going to undo this as per chat I think we should get at least input from @jscook2345 before making that change.

Reply:
That's fair, I'll wait until this becomes unmergeable again :wink: 

The reason why this slipped thru is because while debugging and verifying this I kept running unit tests and they kept failing one at a time. At some point I just edited the Fatal to Skip so that I could get thru with this.